OptimizelySDK/AudienceConditions/BaseCondition.cs

```diff
@@ -68,6 +91,11 @@ public class BaseCondition : ICondition
 
             return evaluator(attributeValue, logger);
         }
+
+        private bool IsValidType()
+        {
+            return new[] {CUSTOM_ATTRIBUTE, THIRD_PARTY_DIMENSION}.Contains(Type);
```

Comment:
every time creating a new object. Make array constant and use `Contains` condition here.

Reply:
This was a great consideration.  Thank you.